docs/helm/upgrades.md

```diff
@@ -21,6 +25,79 @@ Accordingly to this chart rename, also the related values has been moved from
 `.Values.msTeamsService` to
 `.Values.alfresco-connector-msteams`.
 
+### externalHost, externalPort & externalProtocol are not used anymore
+
+In previous versions of the charts one could use the values bellow to tell the
+charts how to configure Alfresco repo & Share, specifically for security
+features (CSRF, CORS):
+
+* `externalHost`
+* `externalPort`
+* `externalProtocol`
+
+These were simple strings and did not allow for multiple domains/URLs to be
+used which is exactly one would be expect to do when configuring cross-origin
+protections.
+This is now possible as we introduce the newer `known_urls` value.
+This value can be iether passed to sshare or repo directly or set in the global
+context. Being able to use either the subchart or the global context Allows for
+conveniently set this parameter at once for all subcharts or on the contrary
+use different values different subcharts if they do not trust the same
+domains/URLs.
+
+global setting:
+
+```yaml
+global:
+  known_urls:
+    - https://ecm.domain.tld
+    - http://app.domain.local:8080/crm
+```
+
+targeted setting:
+
+```yaml
+share:
+  known_urls:
+    - https://ecm.domain.tld
+repository:
+  known_urls:
+    - https://ecm.domain.tld
+    - http://app.domain.local:8080/crm
+```
+
+**The first item in the list will be considered the main domain where alfresco
+& Share are installed**
+
+If you want to use the `--set` switch of the `helm` command`, well first of all
+you should consider stopping doing that and use a local values override file
+instead, otherwise you can still do it by usig either of the syntax below to
+pass a list:
+
+Comma separated list (commas must be escaped):
+
+```bash
+helm install alfresco-content-services acs --set global.known_urls=https://ecm.domain.tld\,http://app.domain.local:8080/crm
+```
+
+Indexed syntax square brackets must be escaped:
+
+```bash
+helm install alfresco-content-services acs \
+  --set global.known_urls\[0\]=https://ecm.domain.tld
+  --set global.known_urls\[1\]=http://app.domain.local:8080/crm
+```
+
+### Alfresco Share chart
+
+Alfresco is now deployed as a separate chart and can be enabled/disabled. In
+order to do so simply use the value bellow:
```

Comment:
```suggestion
Alfresco is now deployed as a separate chart and can be enabled/disabled. In
order to do so simply use the value below:
```

Reply:
Interesting review. I learnt that"bellow" is actually a verb!